app/views/static/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= javascript_pack_tag 'application' %>
+
+<%= react_component("App") %>
```

Comment:
- [ ] You app failed to run. This is what I got while trying to run your app. You might need to check your webpack configuration. Kindly look into it and fix.

![your-app](https://github.com/AbayomiOlaoye/hello-rails-oneapp/assets/26597432/29f76158-90ad-4ea6-9666-9ac29ee75fc3)